src/components/modals/StaticModalComponent.js

```diff
@@ -74,4 +71,55 @@ class StaticModalComponent extends Component<Props> {
   }
 }
 
-export { StaticModalComponent }
+const screenDimensions = {
+  height: Dimensions.get('window').height,
+  width: Dimensions.get('window').width
+}
+
+const styles = {
+  container: {
+    position: 'absolute',
+    justifyContent: 'center',
+    alignItems: 'center',
+    top: (screenDimensions.height * 1) / 8,
+    left: 0,
+    right: 0
+  },
+  modalBox: {
+    paddingHorizontal: screenDimensions.width / 8,
+    width: screenDimensions.width,
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'stretch'
+  },
+  innerBox: {
+    backgroundColor: THEME.COLORS.WHITE
+  },
+  header: {
+    position: 'relative',
+    height: 62,
+    width: '100%',
+    backgroundColor: THEME.COLORS.TRANSPARENT,
+    alignItems: 'center',
+    justifyContent: 'space-around'
+  },
+  bottom: {
+    position: 'relative',
+    width: '100%',
+    backgroundColor: THEME.COLORS.WHITE
+  },
+  bodyRow: {
+    width: '100%',
+    padding: 15
+  },
+  bodyText: {
+    width: '100%',
+    textAlign: 'center',
+    fontFamily: THEME.FONTS.DEFAULT
+  },
+  shim: 20, // Styles.Shim.height,
```

Comment:
These guys mean we can't use `StyleSheet.create`. I could fix them, but this whole component is going to be destroyed soon anyhow.